README.md

```diff
@@ -9,6 +9,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [React India](https://www.reactindia.io/) | 26-28 Sep | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native, GraphQL, VR & AR, Redux, Preact and more. | No |
+| [REDHAT DevConf.IN](https://devconf.info/in) | 3-4 Aug | To be decided | DevConf.IN 2019 is the second annual Developers' Conference to be organised by Red Hat in India. | No |
```

Comment:
The conference is organized by Red Hat but the conference name is just `DevConf.IN`.

Since the conference is free of charge for everybody, can you update the scholarship section to `No (Free for all)`?

Reply:
Also, the date of the conference mentioned on their website is 4-5 Aug, instead of 3-4 Aug.

Reply:
I just checked the website again, it says Aug 2-3 now. Let me fix that and add the scholarship details as well.

Reply:
Can you fix the title as well? To just `DevConf.IN`. :grimacing:

Reply:
Missed that! Done 😬